mteb/tasks/Classification/multilingual/SIB200Classification.py

```diff
@@ -0,0 +1,263 @@
+    "nld": ["nld-Latn"],
+    "nno": ["nno-Latn"],
+    "nob": ["nob-Latn"],
+    "npi": ["npi-Deva"],
+    "nqo": ["nqo-Nkoo"],
+    "nso": ["nso-Latn"],
+    "nus": ["nus-Latn"],
+    "nya": ["nya-Latn"],
+    "oci": ["oci-Latn"],
+    "ory": ["ory-Orya"],
+    "pag": ["pag-Latn"],
+    "pan": ["pan-Guru"],
+    "pap": ["pap-Latn"],
+    "pbt": ["pbt-Arab"],
+    "pes": ["pes-Arab"],
+    "plt": ["plt-Latn"],
+    "pol": ["pol-Latn"],
+    "por": ["por-Latn"],
+    "prs": ["prs-Arab"],
+    "quy": ["quy-Latn"],
+    "ron": ["ron-Latn"],
+    "run": ["run-Latn"],
+    "rus": ["rus-Cyrl"],
+    "sag": ["sag-Latn"],
+    "san": ["san-Deva"],
+    "sat": ["sat-Olck"],
+    "scn": ["scn-Latn"],
+    "shn": ["shn-Mymr"],
+    "sin": ["sin-Sinh"],
+    "slk": ["slk-Latn"],
+    "slv": ["slv-Latn"],
+    "smo": ["smo-Latn"],
+    "sna": ["sna-Latn"],
+    "snd": ["snd-Arab"],
+    "som": ["som-Latn"],
+    "sot": ["sot-Latn"],
+    "spa": ["spa-Latn"],
+    "srd": ["srd-Latn"],
+    "srp": ["srp-Cyrl"],
+    "ssw": ["ssw-Latn"],
+    "sun": ["sun-Latn"],
+    "swe": ["swe-Latn"],
+    "swh": ["swh-Latn"],
+    "szl": ["szl-Latn"],
+    "tam": ["tam-Taml"],
+    "taq": ["taq-Tfng"],
+    "tat": ["tat-Cyrl"],
+    "tel": ["tel-Telu"],
+    "tgk": ["tgk-Cyrl"],
+    "tgl": ["tgl-Latn"],
+    "tha": ["tha-Thai"],
+    "tir": ["tir-Ethi"],
+    "tpi": ["tpi-Latn"],
+    "tsn": ["tsn-Latn"],
+    "tso": ["tso-Latn"],
+    "tuk": ["tuk-Latn"],
+    "tum": ["tum-Latn"],
+    "tur": ["tur-Latn"],
+    "twi": ["twi-Latn"],
+    "tzm": ["tzm-Tfng"],
+    "uig": ["uig-Arab"],
+    "ukr": ["ukr-Cyrl"],
+    "umb": ["umb-Latn"],
+    "urd": ["urd-Arab"],
+    "uzn": ["uzn-Latn"],
+    "vec": ["vec-Latn"],
+    "vie": ["vie-Latn"],
+    "war": ["war-Latn"],
+    "wol": ["wol-Latn"],
+    "xho": ["xho-Latn"],
+    "ydd": ["ydd-Hebr"],
+    "yor": ["yor-Latn"],
+    "yue": ["yue-Hant"],
+    "zho": ["zho-Hant"],
+    "zsm": ["zsm-Latn"],
+    "zul": ["zul-Latn"],
+}
+
+
+def _transform(dataset):
+    dataset = dataset.class_encode_column("category")
+    dataset = dataset.rename_columns({"category": "label"})
+    dataset = dataset.remove_columns(["index_id"])
+    print(len(dataset["test"]))
+    return dataset
+
+
+class SIB200Classification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SIB200Classification",
+        description="""SIB-200 is the largest publicly available topic classification
+        dataset based on Flores-200 covering 205 languages and dialects.
+        """,
+        reference="https://arxiv.org/abs/2309.07445",
+        dataset={
+            "path": "Davlan/sib200",
+            "revision": "38977a667f6fc264d5c26ec57a01e16db040b358",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=_LANGS,
+        main_score="accuracy",
+        date=("2024-01-27", "2024-02-19"),
+        form=["written"],
+        domains=["Constructed"],
```

Comment:
Is the text within the dataset constructed?

Reply:
Good point, the data card says: "The data is sourced from newspaper source and only contains mentions of public figures or individuals", so I'll change it to "News"